packages/material-ui/src/locale/index.js

```diff
@@ -564,6 +610,9 @@ export const frFR = {
 };
 
 export const heIL = {
+  localization: {
+    formatNumber: formatNumber('he-IL'),
+  },
   props: {
     // MuiBreadcrumbs: {
     //   expandText: 'Show path',
```

Comment:
@boazberman Thank you for the initial contribution. We have a couple of new keys, if you have some time, it would be awesome to translate them :).